compose/config.go

```diff
@@ -198,3 +210,12 @@ func (c *Config) GetMinParameterEntropy() int {
 		return c.MinParameterEntropy
 	}
 }
+
+// GetJWTMaxDuration returns the maximum time after JWT issued date, during which the JWT is considered valid.
```

Comment:
Shouldn't this come from the JWT's `exp` field?

Reply:
Well it is calculated as expiration_time - issued_date = token duration. And here we set max allowed duration.

Reply:
After I read through the code I understand that this is another safety measure in cases where the expiry date is set artificially long?

Reply:
yes, you understand it correctly